templates/navigation/menu--external-menu--fallback.html.twig

```diff
@@ -67,7 +67,10 @@
             'mmenu__item',
           ]%}
           <li{{ item.attributes.addClass(item_classes) }}>
-            {% set item_title = item.title %}
+
+            {% set item_title %}
+              <span class="mmenu__link__text"{{ (item.attributes.lang) ? ' lang="' ~ item.attributes.lang ~ '"' : '' }}>{{ item.title }}</span>
```

Comment:
For some reason this `{{ (item.attributes.lang) ? ' lang="' ~ item.attributes.lang ~ '"' : '' }}` renders like so 
![image](https://user-images.githubusercontent.com/1712902/192313474-23f08581-435f-4055-9c68-51b7735ae1f0.png)
If I switch it to `{{ (item.attributes.lang) ? ' lang=' ~ item.attributes.lang : '' }}` then it renders correctly. 
![image](https://user-images.githubusercontent.com/1712902/192313809-3f9cccaa-a3a1-4c4f-b37b-0671861a338b.png)

Can you confirm that this is the case and "it's not just me"? Otherwise this PR can be merged. 🚀